packages/firestore/test/unit/model/target_index_matcher.test.ts

```diff
@@ -956,12 +956,32 @@ describe('Target Bounds', () => {
         ));
     });
 
+    describe('query with multiple inequality', () => {
+      it('returns null', () => {
+        const q = queryWithAddedFilter(
+          queryWithAddedFilter(query('collId'), filter('a', '>=', 1)),
+          filter('b', '<=', 10)
+        );
+        const target = queryToTarget(q);
+        const targetIndexMatcher = new TargetIndexMatcher(target);
+        expect(targetIndexMatcher.hasMultipleInequality()).is.true;
+        const expectedIndex = targetIndexMatcher.buildTargetIndex();
```

Comment:
nit: Rename `expectedIndex` to `actualIndex`, since its value is the _actual_ result, not necessarily the _expected_ result.

Reply:
Please make this change on line 968 as well (i.e. rename `expectedIndex` to `actualIndex`)